vnc.html

```diff
@@ -323,9 +323,9 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
     </audio>
 
     <!-- begin scripts -->
-    <script src="core/util.js"></script>
-    <script src="app/webutil.js"></script>
-    <script src="app/ui.js"></script>
+    <script src="node_modules/browser-es-module-loader/dist/babel-browser-build.js"></script>
+    <script src="node_modules/browser-es-module-loader/dist/browser-es-module-loader.js"></script>
+    <script type="module" src="app/ui.js"></script>
```

Comment:
Yeah, a few of the commits in this PR are not-quite-usable standalone.  Splitting up the commits, like this, IMO, makes the individual commits easier to review and examine historically, since they're broken up by what they do (this commit simply uncomments the module syntax, this commit updates Pako to work with the module syntax, this commit adds in the polyfill, etc).